packages/pigment-css-react/src/baseAtomics.js

```diff
@@ -39,5 +39,57 @@ export const stackAtomics = generateAtomics(({ theme }) => ({
     direction: ['flexDirection'],
     spacing: ['gap'],
   },
-  multiplier: 8,
+  multipliers: {
+    gap: 8,
+  },
+}));
+
+export const gridAtomics = generateAtomics(({ theme }) => ({
```

Comment:
This is same as `stackAtomics`. So you can rename `stackAtomics` to `baseAtomics` and call that for the properties that it has and for the grid specific properties, you can pass those to the `gridAtomics` function. This way, generated classes can be reused.

Reply:
I'll do that 👍🏼 

What about `--Stack-gap`? We should have that in `stackAtomics`, right? We should implement that change in https://github.com/mui/pigment-css/pull/118